dockstore-webservice/src/main/resources/swagger2open.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+# Using https://mermade.org.uk/openapi-converter, generate openapi schema sources.
+STATE=`curl -F "filename=@src/main/resources/swagger.yaml" https://mermade.org.uk/api/v1/convert -o src/main/resources/open_api.yaml -w "%{http_code}"`
+eval "$STATE"
+if [ \( "$?" = 0 \) -o \(  "$STATE" = "200" \) ]; then
```

Comment:
Does this work? I don't know bash very well, so maybe it does. But did you test:

1. Works
2. Fails because curl fails, e.g., the file is missing
3. Fails because the call returns a non-200 value.

Reply:
Basically what I am doing is, executing the command defined by the variable. The block is only true if the curl request has an exit code of 0, and if the response to the http request is a 200. Otherwise it quits with a 0 exit code. 

Methods for testing:
* Swap 200 for a 400 in the logic block, [gave error message] only 200 accepted.
* Swap 0 in exit code of command for 1, [gave error message] only 0 accepted.
* Changed url path to something different, [gave error message] 
* Changed file name, something not there, [gave error message] 
This satisfies the possible conditions of error, and only succeeds if both conditions are satisfied.